climpred/metrics.py

```diff
@@ -171,18 +281,52 @@ def _crpss(forecast, reference, dim='svd', comparison=None):
           Statistical Association 102, no. 477 (March 1, 2007): 359–78.
           https://doi.org/10/c6758w.
 
+    Example:
+        >>> compute_perfect_model(ds, control, metric='crpss')
+        >>> compute_perfect_model(ds, control, metric='crpss', gaussian=False,
+                                  cdf_or_dist=scipy.stats.norm, xmin=-10,
+                                  xmax=10, tol=1e-6)
+
     See also:
         * properscoring.crps_ensemble
+        * xskillscore.crps_ensemble
     """
-    mu = reference.mean(dim)
-    sig = reference.std(dim)
-    ref_skill = _crps_gaussian(forecast, mu, sig, dim=dim)
-    forecast_skill = _crps(forecast, reference, dim=dim)
-    skill_score = (ref_skill - forecast_skill) / ref_skill
+    # available climpred dimensions
+    rdim = [tdim for tdim in reference.dims if tdim in climpred_dims + ['time']]
+    mu = reference.mean(rdim)
+    sig = reference.std(rdim)
+
+    # checking kwargs, if not found use defaults: gaussian, else crps_quadrature
+    if 'gaussian' in kwargs:
+        gaussian = kwargs['gaussian']
+    else:
+        gaussian = True
+    if gaussian:
+        ref_skill = _crps_gaussian(forecast, mu, sig)
+    else:
+        if 'cdf_or_dist' in kwargs:
+            cdf_or_dist = kwargs['cdf_or_dist']
+        else:
+            cdf_or_dist = norm
+        if 'xmin' in kwargs:
+            xmin = kwargs['xmin']
+        else:
+            xmin = None
+        if 'xmax' in kwargs:
+            xmax = kwargs['xmax']
+        else:
+            xmax = None
+        if 'tol' in kwargs:
+            tol = kwargs['tol']
+        else:
+            tol = 1e6
+        ref_skill = _crps_quadrature(forecast, cdf_or_dist, xmin, xmax, tol)
+    forecast_skill = _crps(forecast, reference)
+    skill_score = 1 - forecast_skill / ref_skill.mean('member')
     return skill_score
 
 
-def _less(forecast, reference, dim='svd', comparison=None):
+def _less(forecast, reference, **kwargs):
```

Comment:
I kind of want to rename this function to

log_ens_spread_score since the first thing that pops into my mind when I think of less is the less than sign `<`

Reply:
ok